packages/cozy-stack-client/src/CozyStackClient.spec.js

```diff
@@ -597,6 +597,41 @@ describe('CozyStackClient', () => {
       const newToken = client.refreshToken()
       await expect(newToken).rejects.toThrow()
     })
+
+    fit(`should NOT try to refresh several times in parallele`, async () => {
```

Comment:
nit: in order to fully test this method we may want to test that 2 consecutive calls correctly call the refreshToken 2 times

Reply:
You mean call the `this._promiseCache.exec` 2 times, right?

Reply:
Done